ffi/diplomat/src/script.rs

```diff
@@ -0,0 +1,51 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+#[diplomat::bridge]
+pub mod ffi {
+    use crate::provider::ffi::ICU4XDataProvider;
+    use alloc::boxed::Box;
+    use icu_properties::{provider, script, Script};
+    use icu_provider::DataPayload;
+
+    use crate::errors::ffi::ICU4XError;
+    use diplomat_runtime::DiplomatResult;
+
+    #[diplomat::opaque]
+    /// An ICU4X ScriptWithExtensions map object, capable of holding a map of codepoints to scriptextensions values
+    #[diplomat::rust_link(icu::properties::script::ScriptWithExtensions, Struct)]
+    pub struct ICU4XScriptWithExtensionsSet(
+        pub DataPayload<provider::ScriptWithExtensionsPropertyV1Marker>,
+    );
+
+    impl ICU4XScriptWithExtensionsSet {
```

Comment:
Nit: Misnomer. Just call it `ICU4XScriptWithExtensions`. There's a reason SWE and SES are different in Rust